servers/physics_server_3d.cpp

```diff
@@ -718,7 +755,6 @@ void PhysicsServer3D::_bind_methods() {
 }
 
 PhysicsServer3D::PhysicsServer3D() {
-	ERR_FAIL_COND(singleton != nullptr);
```

Comment:
Is there a change in how servers can be created again? If it's the case,it might be cleaner to have a specific method to reset the singleton.

Reply:
Its the same as the other servers, last one created gets the singleton. The problem is that there wasn't a multithreaded wrapper for the 3D one before so this one had a check.